doc/source/flexibility/innodb_fts_improvements.rst

```diff
@@ -0,0 +1,48 @@
+.. _innodb_fts_improvements:
+
+====================================
+InnoDB Full-Text Search improvements
+====================================
+
+By default n-gram indexes check the `stopwords list
+<https://dev.mysql.com/doc/refman/5.7/en/fulltext-stopwords.html>`_,
+to see if any indexed elements *contain* one of the words on that list.
+
+Using this list for n-gram indexes isn't always suitable, as an example, any
+item that contains ``a`` or ``i`` will be ignored. Another word that can't be
+searched is ``east``, this one will find no matches because ``a`` is on the
+FTS stopword list.
+
+To resolve this issue, in |Percona Server| :rn:`5.7.20-18` a new
+:variable:`innodb_ft_ignore_stopwords` variable has been implemented
+which controls whether |InnoDB| Full-Text Search should ignore stopword list.
+
+System Variables
+================
+
+.. variable:: innodb_ft_ignore_stopwords
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Session, Global
+  :dyn: Yes
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled, this variable will instruct |InnoDB| Full Text Search
+parser to ignore the stopword list when building/updating an FTS index.
+
+.. note::
+
+   Although this variable is introduced to resolve n-gram issues, it affects
```

Comment:
BP description is usually not suitable for copy-pasting into docs directly. This note should be merged to the description above